package.json

```diff
@@ -85,130 +85,128 @@
+    "env-paths": "^2.2.1",
+    "envinfo": "^7.8.1",
+    "execa": "^5.1.1",
+    "express": "^4.17.2",
     "express-logging": "^1.1.1",
     "find-up": "^5.0.0",
     "flush-write-stream": "^2.0.0",
     "folder-walker": "^3.2.0",
     "from2-array": "^0.0.4",
     "fuzzy": "^0.1.3",
-    "get-port": "^5.1.0",
+    "get-port": "^5.1.1",
     "gh-release-fetch": "^3.0.0",
-    "git-repo-info": "^2.1.0",
+    "git-repo-info": "^2.1.1",
     "gitconfiglocal": "^2.1.0",
     "hasbin": "^1.2.3",
     "hasha": "^5.2.2",
-    "http-proxy": "^1.18.0",
-    "http-proxy-middleware": "^2.0.0",
+    "http-proxy": "^1.18.1",
+    "http-proxy-middleware": "^2.0.1",
     "https-proxy-agent": "^5.0.0",
-    "inquirer": "^6.5.1",
-    "inquirer-autocomplete-prompt": "^1.0.1",
-    "is-docker": "^2.0.0",
+    "inquirer": "^8.2.0",
+    "inquirer-autocomplete-prompt": "^1.4.0",
+    "is-docker": "^2.2.1",
     "is-plain-obj": "^3.0.0",
     "is-wsl": "^2.2.0",
     "isexe": "^2.0.0",
-    "jwt-decode": "^3.0.0",
-    "lambda-local": "^2.0.0",
+    "jwt-decode": "^3.1.2",
+    "lambda-local": "^2.0.1",
     "listr": "^0.14.3",
     "locate-path": "^6.0.0",
-    "lodash": "^4.17.20",
-    "log-symbols": "^4.0.0",
+    "lodash": "^4.17.21",
+    "log-symbols": "^4.1.0",
     "log-update": "^5.0.0",
     "memoize-one": "^6.0.0",
     "minimist": "^1.2.5",
-    "multiparty": "^4.2.1",
+    "multiparty": "^4.2.2",
     "netlify": "^10.1.1",
     "netlify-headers-parser": "^6.0.0",
     "netlify-redirect-parser": "^13.0.0",
     "netlify-redirector": "^0.2.1",
-    "node-fetch": "^2.6.0",
+    "node-fetch": "^2.6.6",
     "node-version-alias": "^1.0.1",
     "omit.js": "^2.0.2",
-    "ora": "^5.0.0",
+    "ora": "^5.4.1",
     "p-event": "^4.2.0",
     "p-filter": "^2.1.0",
     "p-map": "^4.0.0",
-    "p-wait-for": "^3.0.0",
+    "p-wait-for": "^3.2.0",
     "parallel-transform": "^1.2.0",
     "parse-github-url": "^1.0.2",
     "parse-gitignore": "^1.0.1",
     "path-key": "^3.1.1",
     "prettyjson": "^1.2.1",
     "pump": "^3.0.0",
-    "raw-body": "^2.4.1",
+    "raw-body": "^2.4.2",
     "read-pkg-up": "^7.0.1",
     "semver": "^7.3.5",
-    "source-map-support": "^0.5.19",
+    "source-map-support": "^0.5.21",
     "static-server": "^2.2.1",
     "string-similarity": "^4.0.4",
     "strip-ansi-control-characters": "^2.0.0",
     "tabtab": "^3.0.2",
-    "tempy": "^1.0.0",
+    "tempy": "^1.0.1",
     "terminal-link": "^2.1.1",
     "through2-filter": "^3.0.0",
     "through2-map": "^3.0.0",
     "to-readable-stream": "^2.1.0",
     "toml": "^3.0.0",
-    "update-notifier": "^5.0.0",
-    "uuid": "^8.0.0",
-    "wait-port": "^0.2.2",
-    "winston": "^3.2.1",
-    "write-file-atomic": "^3.0.0"
+    "update-notifier": "^5.1.0",
+    "uuid": "^8.3.2",
+    "wait-port": "^0.2.9",
+    "winston": "^3.3.3",
+    "write-file-atomic": "^3.0.3"
   },
   "devDependencies": {
-    "@babel/preset-react": "^7.12.13",
-    "@commitlint/cli": "^16.0.0",
```

Comment:
I removed commitlint packages because they are included in `@netlify/eslint-config-node`